scripts/android/build_decred.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+. ./config.sh
+CW_DECRED_DIR=${WORKDIR}/cake_wallet/cw_decred
+LIBWALLET_PATH="${WORKDIR}/decred/libwallet"
+LIBWALLET_URL="https://github.com/itswisdomagain/libwallet.git"
+
+mkdir -p $LIBWALLET_PATH
+git clone $LIBWALLET_URL $LIBWALLET_PATH --branch cgo
+cd $LIBWALLET_PATH
```

Comment:
Can remove if exists, otherwise I think cloning will fail. Better to clone anew every time in case of changes?

```suggestion

if [ -e $LIBWALLET_PATH ]; then
       rm -fr $LIBWALLET_PATH
fi
mkdir -p $LIBWALLET_PATH
git clone $LIBWALLET_URL $LIBWALLET_PATH --branch cgo
cd $LIBWALLET_PATH
```